src/eigen/lobpcg_hyper_impl.jl

```diff
@@ -210,11 +210,13 @@ function ortho(X, Y, BY; tol=2eps(real(eltype(X))))
 end
 
 
-function final_residnorms(A, X, resids, niter)
+function final_residnorms(A, X, resids, niter, n_matvec)
     AX = A * X
+    n_matvec += size(X, 2)
```

Comment:
don't forget to remove this when merging against master

Reply:
Sure ... for now I'm running some bigger tests on this branch so I'm not merging with master to stay comparable in the numbers, but I have it in mind.